Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```
@@ -33,63 +33,88 @@
 
 - type: entity
   parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterCoatToggleable
+  name: winter coat with hood
+  noSpawn: True
+  components:
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadHatHoodWinterDefault
+    slot: head
+  - type: ContainerContainer
+    containers:
+      toggleable-clothing: !type:ContainerSlot {}

```

```suggestion
  - type: ContainerContainer
    containers:
      toggleable-clothing: !type:ContainerSlot {}
      storagebase: !type:Container
        ents: []
```

You're overwriting the containercontainer component from the ClothingOuterStorageBase. This fixes test fails.

---

Thank you so much